tests/modules/test_nat.py

```diff
@@ -0,0 +1,59 @@
+import unittest
+import tempfile
+import yaml
+from genisys.modules.nat import Nat
+from genisys import configParser
+
+class NatTest(unittest.TestCase):
+    """ Tests for the NAT module """
+    def test_values_in_config(self):
+        """ Ensure that a missing config option raises the expected error. """
+        with open("../configs/nat_test_1.yml") as config_file:
```

Comment:
Make sure this runs when you are in the project root dir, ie `poetry run python tests/modules/test_nat.py`. The GitHub action runs from the project root and it is my understanding that paths in python are relative to the current working directory. In the past I've had to do shenanigans to get the current file's directory like `os.path.abspath(os.path.dirname(__file__))` then append your relative file name from there. Alternatively, always run tests from the project root and convert these pathnames to `tests/configs/...`